ibm-mq-mixin/panels.libsonnet

```diff
@@ -0,0 +1,589 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+
+{
+  new(this)::
+    {
+      local signals = this.signals,
+
+      /*
+      -------------------------
+      Cluster Overview
+      -------------------------
+      */
+
+      clusterClusters:
+        commonlib.panels.generic.stat.info.new('Clusters', targets=[signals.cluster.clusters.asTarget()])
+        + g.panel.stat.panelOptions.withDescription('The unique number of clusters being reported.')
+        + g.panel.stat.standardOptions.withUnit('none'),
+
+      clusterQueueManagers:
+        commonlib.panels.generic.stat.info.new('Queue managers', targets=[signals.cluster.queueManagers.asTarget()])
+        + g.panel.stat.panelOptions.withDescription('The unique number of queue managers in the cluster being reported.')
+        + g.panel.stat.standardOptions.withUnit('none'),
+
+      clusterTopics:
+        commonlib.panels.generic.stat.info.new('Topics', targets=[signals.cluster.topics.asTarget()])
+        + g.panel.stat.panelOptions.withDescription('The unique number of topics in the cluster.')
+        + g.panel.stat.standardOptions.withUnit('none'),
```

Comment:
Couple things here:

We'll probably want to have a "no data" override set to "0"

Additionally, we should use `short` for basic numbers instead of `none` for units, lest the linter complain :)